kie-server-parent/kie-server-router/kie-server-router-proxy/src/main/java/org/kie/server/router/handlers/AdminHttpHandler.java

```diff
@@ -45,16 +46,10 @@
 import io.undertow.server.handlers.ResponseCodeHandler;
 import io.undertow.util.Headers;
 
-import static org.kie.server.router.KieServerRouterConstants.*;
-
 public class AdminHttpHandler implements HttpHandler {
     
     private static final Logger log = Logger.getLogger(AdminHttpHandler.class);
 
-    private String CONTROLLER = System.getProperty(KIE_CONTROLLER);
-    private int interval = Integer.parseInt(System.getProperty(KIE_SERVER_CONTROLLER_ATTEMPT_INTERVAL, "5"));
-    private int attemptsLimit = Integer.parseInt(System.getProperty(KIE_SERVER_RECOVERY_ATTEMPT_LIMIT, "100"));
-    
 //    private KieServerProxyClient proxyClient;
```

Comment:
You could remove also this commented line

Reply:
removed